internal/graph/check.go

```diff
@@ -1408,10 +1408,23 @@ func (c *LocalChecker) checkTTU(parentctx context.Context, req *ResolveCheckRequ
 			}
 		}
 
-		if typesys.RecursiveTTUCanFastPath(objectTypeRelation, userType) {
+		choice1 := typesys.RecursiveTTUCanFastPath(objectTypeRelation, userType)
```

Comment:
Whats going on here? Guessing this is you testing? Because we shouldn't attempt to apply these algorithms like this.

Reply:
Could you be more specific? What is "like this"?

I wanted to make it so that if v2 applies, and the experimental is enabled, it will supersede v1.

Reply:
Amended